be/src/olap/tablet_manager.cpp

```diff
@@ -616,19 +615,12 @@ bool TabletManager::get_rowset_id_from_path(const string& path, RowsetId* rowset
 }
 
 void TabletManager::get_tablet_stat(TTabletStatResult* result) {
-    int64_t curr_ms = UnixMillis();
-    // Update cache if it is too old
+    std::shared_ptr<std::vector<TTabletStat>> local_cache;
```

Comment:
Why did you remove the update logic?

Reply:
it is Inefficient，we can produce cache in build_tablet_info function